src/shared/hooks/useWindowSize.test.ts

```diff
@@ -0,0 +1,31 @@
+import { act, renderHook } from '@testing-library/react';
+
+import { useWindowSize } from './useWindowSize';
+
+describe('useWindowSize', () => {
+  test('should return correct height and width for window', () => {
+    const { result } = renderHook(() => useWindowSize());
+
+    expect(result.current.width).toEqual(window.innerWidth);
+    expect(result.current.height).toEqual(window.innerHeight);
+  });
+
+  const windowWidth = window.innerWidth;
```

Comment:
maybe these variables can be used also for previous test case, so it make sense to move them up?

Reply:
fixed